tests/test_runs/test_run_functions.py

```diff
@@ -1453,7 +1511,24 @@ def test_format_prediction_task_learning_curve_sample_not_set(self):
             format_prediction(learning_curve, *ignored_input, sample=None, proba=probabilities)
 
     def test_format_prediction_task_regression(self):
-        regression = openml.tasks.get_task(self.TEST_SERVER_TASK_REGRESSION[0], download_data=False)
+        task_id = self.TEST_SERVER_TASK_REGRESSION[0]
```

Comment:
Is there a reason why you only needed to update three tests? Are the other tests relying on the 'more complicated' datasets from the OpenML100?